src/parseTools.js

```diff
@@ -1253,3 +1253,8 @@ function makeMalloc(source, param) {
   }
   return `abort('malloc was not included, but is needed in ${source}. Adding "_malloc" to EXPORTED_FUNCTIONS should fix that. This may be a bug in the compiler, please file an issue.');`
 }
+
+function getDynCaller(sig) {
```

Comment:
Given that this only has one logical call site how about just inlining it in library.js:

```
#if MINIMAL_RUNTIME
var f = dynCalls[sig];
#else
var f = Module["dynCall_" + sig];
#endif
if (args && args.length) {
  return f.apply(null, [ptr].concat(args));
}
return f.call(null, ptr);
```

Reply:
sure